src/Configuration/Cache.php

```diff
@@ -11,12 +11,15 @@
 
 namespace Sensio\Bundle\FrameworkExtraBundle\Configuration;
 
+use Attribute;
```

Comment:
We don't import classes from the global namespace.

Reply:
I agree but fabbot.io fail if I don't do that :-/

Reply:
fabbot is wrong :)